### PR TITLE
Add Validate Operator Bundle check to CraneEngine

### PR DESCRIPTION
**Pull request by bcrochet**
The Validate Operator Bundle will now look for a bundle path, instead of a bundle image.

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

**Review by jomkz — APPROVED** [review]
/lgtm

**Review by acornett21 — APPROVED** [review]

**Comment by bcrochet** [comment]
I will validate that this does not require privilege before merging.

/hold

**Comment by bcrochet** [comment]
/hold cancel

This is passing without privilege now.

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:62]: You could merge these two lines into one.

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:78]: same goes here

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:85]: same goes here

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:141]: This could be moved to `util/file` for future reuse.

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:180]: I am wondering If the tempConfigFile has been created successfully, but the function fails to run `WriteString`, do we want to return tempConfigFile as non-nil value along with a non-nil error?

**Review by samira-barouti — COMMENTED** [review]
A few nits and a question regarding the error handling

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [reply certification/engine/operatorsdk.go:180]: nvm; Found out how the caller is making use of both return values.

**Review by acornett21 — APPROVED** [review]

File: certification/engine/operatorsdk.go
Comment (samira-barouti) [inline certification/engine/operatorsdk.go:50]: This defer statement could be moved to line 46 to ensure the configFile is removed in the case of errors.

**Review by samira-barouti — APPROVED** [review]
/lgtm

**Pull request merged**
